app/src/main/java/org/oppia/android/app/testing/DragDropTestFragment.kt

```diff
@@ -0,0 +1,35 @@
+package org.oppia.android.app.testing
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.android.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+const val DRAG_DROP_TEST_FRAGMENT_TAG = "drag_drop_adapter_test_fragment"
+
+class DragDropTestFragment : InjectableFragment() {
```

Comment:
Please add a KDoc briefly explaining what this is/what it's for. Ditto for the const above & elsewhere in the PR (all non-private, non-override things should be documented).